portfolio/src/main/webapp/index.html

```diff
@@ -12,63 +12,71 @@
-                    <div class="gallery">
+                <div class=main-body>
+                    <div class="mini-container brief">
+                        <h2>
+                            <br>Kizito Akwotiga</br>
+                            <br>Computer Enginering Student</br>
+                            <address>kakwotiga@gmail.com</address>
+                        </h2>
+                        <img class="profile-picture" src="images/AtWork.jpg">
+                    </div>
+                    <div class="mini-container">
+                        I am Kizito Akwotiga, a second year computer engineering student at City University of New
+                        York-QCC,
+                        very much interested in software development and looking forward to bulding things for the web.
+                        I find electrical circuits satisfying which led me to pursueing a degree in Engineering.
+                        This is my very first portfolio and hopefully it will get better before SPS comes to an end.
+                    </div>
+                    <div class="mini-container">
+                        I have worked with a group of software developers to create an E-Commerce website. As a
+                        participant,
+                        I sole handedly worked on the GUI of the login page and loved the outcome. You may visit the <a
+                            href="https://github.com/ayamkito/qcc-shopper.git">repository </a> and leave any feed back.
+                    </div>
+                    <div class="mini-container" id="social">
+                        You may connect with me on professional/social media:
                         <div>
-                            <button onclick="prevImage"> << </button>
-                            <img src="https://globalgrasshopper.com/wp-content/uploads/2010/03/Kruger-National-Park-1000x664.jpg"
-                                id="gallery">
-                            <button onclick="nextImage"> >> </button>
+                            <a href="https://github.com/ayamkito">github</a>
+                            <a href="https://www.linkedin.com/in/ayamkito/">LinkedIn</a>
+                            <a href="#">Instagram</a>
                         </div>
-                        <p>The heart of Africa lies in it's people and it's wild life.</p>
                     </div>
-                </div>
-                <div class="flexWrap-container-reg">
-                    As this portfolio gets better, I will dedicate this space for the stories of the wild.
+                    <div class="mini-container">
+                        <div class="gallery">
+                            <div>
+                                <button onclick="prevImage">
+                                    << </button>
+                                        <!-- <img src="https://globalgrasshopper.com/wp-content/uploads/2010/03/Kruger-National-Park-1000x664.jpg" -->
```

Comment:
This seems like this isn't correctly commented. You may want to comment the next line as well.

But more generally, as a rule of thumb, before committing any code, you should remove any unused commented code and only keep explanatory comments.